src/components/task/TaskPanel.tsx

```diff
@@ -0,0 +1,186 @@
+import { Card, CardActions, CardContent, Collapse, ExpansionPanelDetails, ExpansionPanelSummary, Grid, IconButton, Typography } from '@material-ui/core';
+import MuiExpansionPanel from '@material-ui/core/ExpansionPanel';
+import ExpandLessIcon from '@material-ui/icons/ExpandLess';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import { withStyles } from '@material-ui/styles';
+import { APITask, APITest, ConvertToAPITests } from 'evergreen.js/lib/models';
+import * as React from 'react';
+import * as rest from "../../rest/interface";
+import '../../styles.css';
+
+export const StyledExpansionPanel = withStyles({
+  root: {
+    border: '1px solid rgba(0, 0, 0, .125)',
+    boxShadow: "none"
+  }
+})(MuiExpansionPanel);
+
+interface State {
+  hasFailingTests: boolean
```

Comment:
Do we need these 2 boolean properties? Would checking the length of the arrays work instead?

Reply:
No, we don't really need them, I just thought it was cleaner. I can swap these properties out for length checks in my code.

Reply:
Fixed